homework-submission/homework-week3/react-hyf/server/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "concurrently": "^4.1.0",
```

Comment:
For an easier use you may have an external `package.json` in the parent directory runs both client and server with concurrently, and it happen you don't need it here in the server directory but in the parent :)